webpack.common.js

```diff
@@ -35,7 +36,9 @@ module.exports = (env = {}) => ({
           options: {
             plugins: [
               // TODO: Add necessary remark plugins
+              require('remark-slug'),
               require('remark-autolink-headings'),
+              require('remark-html'),
```

Comment:
`remark-html` should be dropped as the `remark-loader` already includes it at the end of the chain.